test/reconstruction/gapfill.jl

```diff
@@ -0,0 +1,54 @@
+@testset "Gapfill" begin
+    #=
+    Implement the small model that should be gapfilled.
+    =#
+    model = StandardModel("partialmodel")
+    m1 = Metabolite("m1")
+    m2 = Metabolite("m2")
+    m3 = Metabolite("m3")
+    m4 = Metabolite("m4")
+    m5 = Metabolite("m5")
+    m6 = Metabolite("m6")
+    m7 = Metabolite("m7")
+    m8 = Metabolite("m8")
```

Comment:
cycle/array?

Reply:
btw wouldn't it be simpler to take some of the toy models, literally remove the reactions from it and see if this thing can add them back?

Reply:
(even e_coli_core :] )

Reply:
will make another test, this is nice to troubleshoot due to size

Reply:
actually, not doing this, MILP takes long to solve for larger models, better to test on this tiny one

Reply:
also, I need the metabolites as variables when constructing the reactions, so there is no array possibility